main/eval/src/mill/eval/EvaluatorCore.scala

```diff
@@ -32,7 +32,7 @@ private[mill] trait EvaluatorCore extends GroupEvaluator {
       testReporter: TestReporter = DummyTestReporter,
       logger: ColorLogger = baseLogger,
       serialCommandExec: Boolean = false
-  ): Evaluator.Results = {
+  ): Evaluator.Results = logger.withPromptUnpaused {
```

Comment:
This looks unrelated to deprecations removal. Should be a separate PR.

Reply:
It's part of cleaning up the override in `EvaluatorImpl` which was marked for removal

```scala
  override def evaluate(
      goals: Strict.Agg[Task[_]],
      reporter: Int => Option[CompileProblemReporter],
      testReporter: TestReporter,
      logger: ColorLogger,
      serialCommandExec: Boolean
  ): Evaluator.Results = {
    // TODO: cleanup once we break bin-compat in Mill 0.13
    // disambiguate override hierarchy
    logger.withPromptUnpaused {
      super.evaluate(goals, reporter, testReporter, logger, serialCommandExec)
    }
  }
```

Reply:
Should be a no-op change

Reply:
Understood. (I commented before I fully saw all changes in detail)